modules/oadp-gcp-wif-cloud-authentication.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-gcp-wif-cloud-authentication_{context}"]
+= Google WIF cloud authentication
+
+Applications running outside Google Cloud use service account keys to access Google Cloud resources, these service account keys can become a security risk if they are not managed correctly.
+
+With identity federation (WIF), you can use Identity and Access Management (IAM) to grant external identities IAM roles, including the ability to impersonate service accounts. This eliminates the maintenance and security risks associated with service account keys.
+
+WIF handles encrypting and decrypting certificates, extracting user attributes, and validation. WIF externalizes authentication, passing it over to Security Token Services (STS) and reduces the demands on individual developers. Authorization and controlling access to resources remain the responsibility of the application.
+
+[NOTE]
+====
+Google WIF cloud authentication is avail for OADP 1.3.x and later.
```

Comment:
```suggestion
Google WIF cloud authentication is available for OADP 1.3.x and later.
```

There are just a few little mistakelets in this doc, please just have a read-through